client/signup/steps/woocommerce-install/transfer/transfer-site.tsx

```diff
@@ -45,15 +57,16 @@ export default function TransferSite(): ReactElement | null {
 		() => {
 			dispatch( requestLatestAtomicTransfer( siteId ) );
 		},
-		transferStatus === transferStates.COMPLETED ? null : 3000
+		transferFailed || transferStatus === transferStates.COMPLETED ? null : 3000
```

Comment:
If there is an old failure before the new transfer is kicked off this will never poll. I think a timeout that blocks this after x seconds is a better option.